corehq/apps/app_manager/templates/app_manager/form_view.html

```diff
@@ -166,6 +165,38 @@
     <div class="tab-content appmanager-tab-content">
       {% if form.uses_cases %}
         <div class="tab-pane" id="case-configuration">
+          {% if case_property_warning.show %}
+            <div class="alert alert-warning">
+              <button
```

Comment:
Wondering if only the small button (#performance-warning-toggle) is clickable? I feel make the entire header line clickable is more user-friendly.

Reply:
Yeah only the button is clickable. I'd be curious to hear your feedback after playing with it on staging.